gatsby-config.js

```diff
@@ -45,12 +53,43 @@ module.exports = {
               maxWidth: 750,
             },
           },
+          'gatsby-remark-autolink-headers',
           `gatsby-remark-prismjs`,
+          'gatsby-remark-copy-linked-files',
+        ],
+      },
+    },
+    //
+    // We need this "duplicate" config here, due to
+    // https://github.com/gatsbyjs/gatsby/issues/19859
```

Comment:
After reading https://www.gatsbyjs.org/packages/gatsby-plugin-mdx/#gatsby-remark-plugins, I think this sounds like a bug.  It's not expected behavior. I will go back into the issue and bring up possibility of making a new bug issue